### PR TITLE
Fortinet new CVEs

### PR DESCRIPTION
**Pull request by jimzengca**
Add 3 new CVEs for Fortinet.

### CODE REVIEW

File: 2017/7xxx/CVE-2017-7335.json [hunk 1]
```diff
@@ -1,18 +1,57 @@
...
+      {
+        "url": "https://fortiguard.com/psirt/FG-IR-17-106"
+      }
+    ]
+  },
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "A Cross-Site Scripting (XSS) vulnerability in Fortinet FortiWLC allows an authenticated user to inject arbitrary web script or HTML via non-sanitized parameters "refresh" and "branchtotable" present in HTTP POST requests."
```
Comment (cve-team) [inline 2017/7xxx/CVE-2017-7335.json]: Double quotes within a string need to be escaped.

**Comment by cve-team** [comment]
The files are missing the 'data_type', 'data_format', and 'data_version' properties, each of which is required.

File: 2017/7xxx/CVE-2017-7732.json [hunk 2]
```diff
@@ -1,18 +1,57 @@
...
-   }
+    }
+  },
+  "problemtype": {
+    "problemtype_data": [
+      {
+        "description": [
+          {
+            "lang": "eng",
+            "value": "Execute unauthorized code or commands"
```
Comment (cve-team) [inline 2017/7xxx/CVE-2017-7732.json]: Is this correct? The underlying issue is an XSS vulnerability.

File: 2017/7xxx/CVE-2017-7335.json [hunk 3]
```diff
@@ -1,18 +1,57 @@
...
-   }
+    }
+  },
+  "problemtype": {
+    "problemtype_data": [
+      {
+        "description": [
+          {
+            "lang": "eng",
+            "value": "Execute unauthorized code or commands"
```
Comment (cve-team) [inline 2017/7xxx/CVE-2017-7335.json]: Is this correct? The underlying issue is an XSS vulnerability.

**Comment by jimzengca** [comment]
Updated json files per feedback.

**Comment by cve-team** [comment]
Thanks, look for the entries to be published on https://cve.mitre.org within the next few hours.

George

**Pull request merged**